src/algokit_utils/_transfer.py

```diff
@@ -37,7 +35,26 @@ class TransferParameters:
     if this is set it's possible the transaction could get rejected during network congestion"""
 
 
-def _check_fee(transaction: PaymentTxn, max_fee: int | None) -> None:
+@dataclasses.dataclass(kw_only=True)
+class TransferParameters(TransferParametersBase):
+    # """Parameters for transferring µALGOs between accounts"""
+
+    micro_algos: int
+
+
+@dataclasses.dataclass(kw_only=True)
+class TransferAssetParameters(TransferParametersBase):
+    """Parameters for transferring µALGOs between accounts"""
+
+    index: int
+    """The asset id that will be transfered"""
+    amt: int
```

Comment:
Suggest we use `amout` rather than `amt` so it's clearer, and we should indicate in the doc comment that it's in the decimal unit.